src/components/SketchFabViewer.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="sketchfab-viewer image is-16by9">
+  <div class="sketchfab-viewer image is-sketchfab-viewer">
```

Comment:
```suggestion
  <div class="sketchfab-viewer image is-16by9">
```

I think it's best to keep the original class name.